tools/depends/target/Makefile

```diff
@@ -9,7 +9,6 @@ DEPENDS = \
   brotli \
   bzip2 \
   dav1d \
-  exiv2 \
```

Comment:
Care to explain why it's not needed here?

Reply:
When an internal build is available, just let it handle it.

You can still manually trigger using make exiv2 if desired